Google.PowerShell/BigQuery/BqCmdlet.cs

```diff
@@ -19,10 +19,15 @@ public class BqCmdlet : GCloudCmdlet
         public BigqueryService Service => _service.Value;
         public BigQueryClient Client => _client.Value;
 
+        internal static BigqueryService DefaultBigQueryService { private get; set; }
```

Comment:
The naming on these seems a little odd to me.  I think default normally implies if nothing else is provided then use the default. 

These can't ever be overridden by a user using the cmdlts right?  Maybe just have the old ones be the default and override them?